.github/workflows/npm-publish.yml

```
@@ -13,7 +13,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: '18'
+          node-version: '20'
           registry-url: 'https://registry.npmjs.org'
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)